Engine/Module/ModuleRender.cpp

```diff
@@ -227,6 +228,8 @@ void ModuleRender::RenderFrame(const ComponentCamera &camera)
 	}
 	glDisable(GL_BLEND);
 	
+	BROFILER_CATEGORY("Canvas", Profiler::Color::AliceBlue);
+	App->ui->Render(&camera);
```

Comment:
UI is rendered in the method `RecordFrame`, adding this line makes it render twice

Reply:
I haven't coded this line..hmm..i'll delete it!